server.ts

```diff
@@ -1052,6 +1087,14 @@ app.post("/job/:jobId/submit", async function (req, res) {
   res.json(Helper.job2object(job));
 });
 
+/**
+ * @openapi
+ * /clean:
+ *  put:
+ *      description: Not yet implemented
+*/
+app.put("/clean", async function (req, res) {});
```

Comment:
It seems like we can remove this from the PR since it doesn't actually implement anything anyways.